data/card_catalogue.json

```diff
@@ -123,27 +215,17 @@
         {
           "title": "Empouched Earth",
           "upgrade": "",
-          "tier": "grey",
+          "tier": "blue",
           "details": {
             "rules": "One handful of pocket sand.",
             "flavour": "Good for blinding a close range target",
             "image": "assets/core/card/illustration/Empouched Earth.png"
           }
         },
-        {
-          "title": "The Ship-Christener",
-          "upgrade": "",
-          "tier": "green",
-          "details": {
-            "rules": "Shake the bottle and return this card to the deck after picking one: Fire the cork, smash on a character, enjoy with allies. ",
-            "flavour": "Cheers",
-            "image": "assets/core/card/illustration/The Ship-Christener.png"
-          }
-        },
```

Comment:
1. I'm trying to avoid `return this card to the deck` in rules text\
2. I think the items which we introduced later cover what this did previously